easybuild/easyconfigs/p/pybind11/pybind11-2.10.3-GCCcore-12.2.0.eb

```diff
@@ -9,15 +9,24 @@ toolchain = {'name': 'GCCcore', 'version': '12.2.0'}
 
 source_urls = ['https://github.com/pybind/pybind11/archive/']
 sources = ['v%(version)s.tar.gz']
-checksums = ['5d8c4c5dda428d3a944ba3d2a5212cb988c2fae4670d58075a5a49075a6ca315']
+patches = [
+    'pybind11-2.10.3_fix-nvcc-compat.patch',
+    'pybind11-2.10.3_require-catch.patch',
+]
+checksums = [
+    {'v2.10.3.tar.gz': '5d8c4c5dda428d3a944ba3d2a5212cb988c2fae4670d58075a5a49075a6ca315'},
+    {'pybind11-2.10.3_fix-nvcc-compat.patch': '510a23dac47b8b440c06c101d269451c95e09907d9034b6b8a16aeb8b89364ae'},
+    {'pybind11-2.10.3_require-catch.patch': '4a27ba3ef1d5c535d120d6178a6e876ae678e4899a07500aab37908357b0b60b'},
+]
 
 builddependencies = [
     ('binutils', '2.39'),
     ('CMake', '3.24.3'),
+    # Test dependencies
     ('Eigen', '3.4.0'),
+    ('Catch2', '2.13.9', '', SYSTEM),
 ]
-dependencies = [('Python', '3.10.8')]
 
-configopts = "-DPYTHON_EXECUTABLE=$EBROOTPYTHON/bin/python"
```

Comment:
was moving this line completely from only python11-2.10.3 intentional?

Reply:
It was supposed to get removed from both, but missed the 2nd. This is done in the easyblock so we don't need to copy this around to all ECs. Pushed an update